Makefile

```diff
@@ -21,8 +21,8 @@ maintainer-clean: clean
 docs:
 	$(MAKE) -C docs clean man html
 
-update-docs: install
-	bash ./docs/update-help-messages.sh
+update-docs: bin
+	PGCOPYDB=${TOP}src/bin/pgcopydb/pgcopydb bash ./docs/update-help-messages.sh
```

Comment:
Please define the PGCOPYDB make variable earlier in the file in a way that it can be overloaded from the outside, typically:

```
PGCOPYDB ?= $(TOP)src/bin/pgcopydb/pgcopydb
```